apps/emqx_management/src/emqx_mgmt_cli.erl

```diff
@@ -99,18 +99,18 @@ broker(_) ->
 cluster(["join", SNode]) ->
     case ekka:join(ekka_node:parse_name(SNode)) of
         ok ->
-            emqx_ctl:print("Join the cluster successfully.~n"),
+            emqx_ctl:print("Join the cluster successfully.~n", []),
```

Comment:
do we have any reason to keep `emqx_ctl:print/1`?

Reply:
Good question.  Perhaps a better solution would be to change it to always include the empty args list for the format.  Then, one can use it without this awkward argument.

(But the reason to use `emqx_ctl:print/1` is convenience, I'd guess)

I'll change it.

Reply:
maybe no need to keep `format/1` then ?